src/abhishekSJun20/PrimeNumber.java

```diff
@@ -0,0 +1,66 @@
+package abhishekSJun20;
+
+/*
+ 1: Create a class to satisfy below requirements. 
+    a) Given number is prime or not. 
+    b) Print total count of prime numbers between 2->100.
+    c) Find sum of all prime numbers between 2->100
+    d) Find avg of prime numbers between 2->100
+
+ */
+public class PrimeNumber {
+
+	int primeNumber(int num) {
+		// Method to find if Number is Prime Number or not. It return num - If prime and
+		// returns 0 - If not Prime
+		for (int index = 2; index <= Math.sqrt(num); index++) {
+			if (num % index == 0) {
+				return 0;
+			}
+		}
+		return num;
+	}
+
+	int countOfPrimeNumInRange(int startRange, int endRange) {
+		// Method to find count of Prime number in given range. Is call primeNumber
+		// Method in a loop to check the individual elements
+		// Method displays and return count of all the prime number in the range
+		int count = 0;
+		for (int index = startRange; index <= endRange; index++) {
+			if (primeNumber(index) > 0) {
+				count++;
+			}
+		}
+		System.out.println("Count of Prime Number between range :" + count);
+		return count;
+	}
+
+	int sumAllPrimeNumInRange(int startRange, int endRange) {
+		// Method to calculate sum of Prime numbers in given range. Is call primeNumber
+		// Method in a loop to check the individual elements
+		// Method displays and return sum of all the prime number in the range
+		int sum = 0;
+		for (int index = startRange; index <= endRange; index++) {
+			sum += primeNumber(index);
+		}
+		System.out.println("Sum of Prime Number between range :" + sum);
+		return sum;
+	}
+
+	double AverageAllPrimeNumInRange(int sumOfPrimeNum, int countOfPrimeNum) {
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
verifyNumberIsPrimeOrNot2(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);